modules/SofaUserInteraction/src/SofaUserInteraction/RayTraceDetection.cpp

```diff
@@ -240,10 +245,8 @@ void RayTraceDetection::addCollisionModel (core::CollisionModel * cm)
 {
     if (cm->empty ())
         return;
-    for (sofa::helper::vector < core::CollisionModel * >::iterator it =
-            collisionModels.begin (); it != collisionModels.end (); ++it)
+    for (auto* cm2 : collisionModels)
```

Comment:
auto* will not be deduced as core::CollisionModel** ? (as collisionModels is a vector of CollisionModel*)

Reply:
No, I don't think so. See https://stackoverflow.com/a/34560387

Reply:
👍